app/controllers/miq_ae_tools_controller.rb

```diff
@@ -395,7 +395,7 @@ def get_form_vars
       @resolve[:new][:target_class] = params[:target_class]
       klass = CustomButton.name_to_button_class(params[:target_class])
       unless klass.nil?
-        targets = klass.all
+        targets = klass
```

Comment:
Agreed, I think the `name_to_button_class` is not used and should be deleted.

@kbrock Thanks for the Rbac suggestions.